plugins/marvin.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: marvin
+spec:
+  homepage: https://github.com/undistro/marvin
+  shortDescription: Scans your cluster for potential issues. Supports custom checks with CEL.
```

Comment:
Shorten to 50 chars as it gets truncated otherwise.

Reply:
Ok, I've shorten it. Thanks.